jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -814,88 +814,77 @@ def _fft_rng_factory(dtype):
   for window_dilation in [(1, 1, 1, 1)]
 )
 
-lax_reduce_window = tuple(
-  # Tests with 2d shapes (see tests.lax_test.testReduceWindow)
-  Harness(f"2d_shape={jtu.format_shape_dtype_string(shape, dtype)}_initvalue={init_value}_computation={computation.__name__}_windowdimensions={window_dimensions}_windowstrides={window_strides}_padding={padding}_basedilation={base_dilation}_windowdilation={window_dilation}".replace(' ', ''),
-          lax.reduce_window,
-          [RandArg(shape, dtype), StaticArg(init_value), StaticArg(computation),
-           StaticArg(window_dimensions), StaticArg(window_strides),
-           StaticArg(padding), StaticArg(base_dilation), StaticArg(window_dilation)],
-          shape=shape,
-          dtype=dtype,
-          init_value=init_value,
-          computation=computation,
-          window_dimensions=window_dimensions,
-          window_strides=window_strides,
-          padding=padding,
-          base_dilation=base_dilation,
-          window_dilation=window_dilation)
-  for computation in [lax.add, lax.max, lax.min, lax.mul]
-  for dtype in { lax.add: filter(lambda t: t != np.bool_, jtu.dtypes.all)
-               , lax.mul: filter(lambda t: t != np.bool_, jtu.dtypes.all)
```

Comment:
I meant to point out that I have never see this formatting style, with the lines starting with commas. It is a minor matter, but it does seem annoying. I think that there is no "information content" in the comma and seeing it first in the line seems distracting.

Reply:
It's quite common in Haskell code, but it's true I haven't really seen it elsewhere! I find it personally more readable, and it makes it easier to add/remove lines – also the diff contains less lines when doing so, since we don't need to add/remove a comma on the previous line when doing that. Nevertheless, noted :)